### PR TITLE
feat(docs-design): refine core styles

### PR DESCRIPTION
**Pull request by dagoof**

### CODE REVIEW

File: docs/src/components/sidenav.js [hunk 1]
```diff
@@ -1,18 +1,39 @@
...
 
-const NavItem = styled.li``
+const SubNav = styled.ul`
+  list-style: none;
+  margin: 0;
+  padding: 0;
+`
 
 const Nav = props => (
   <NavWrapper {...props}>
```
Comment (andrewoh531) [inline docs/src/components/sidenav.js]: Out of curiosity, what's the purpose of passing `props` into `NavWrapper`?

**Comment by jonathaningram** [comment]
I just submitted my pending comments, must have forgotten to press submit previously. Happy to just proceed with this and iterate on it.

File: docs/src/components/code.js [hunk 2]
```diff
@@ -34,7 +33,12 @@ export default props => (
...
       <SyntaxHighligher
         style={tomorrowNight}
-        customStyle={{ background: 'none', margin: '0', padding: '0' }}
+        customStyle={{
+          background: 'none',
+          margin: '0',
+          padding: '0',
+          overflowX: 'scroll',
+        }}
         {...props}
```
[the remark this answers is not in the document]
Comment (jonathaningram) [reply docs/src/components/code.js]: @dagoof ?

File: docs/src/components/sidenav.js (hunk 1 above)
Comment (jonathaningram) [reply docs/src/components/sidenav.js]: @dagoof ?

**Pull request merged**